src/main/resources/env.schema.json

```diff
@@ -1,120 +1,120 @@
       "description": "For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server."
     },
     "AWS_API_STAGE": {
       "type": "string",
-      "default": "stage-name",
+      "examples": ["stage-name"],
       "description": "For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server."
     },
     "BUGSNAG_API_KEY": {
       "type": "string",
-      "default": "bugsnag-api-key",
+      "examples": ["bugsnag-api-key"],
       "description": "A valid Bugsnag authorization token."
     },
     "BUGSNAG_EVENT_JOB_DELAY_IN_MINUTES": {
       "type": "integer",
-      "default": "1",
+      "examples": ["1"],
       "description": "Bugsnag event job frequency."
     },
     "BUGSNAG_EVENT_REQUEST_JOB_DELAY_IN_MINUTES": {
       "type": "integer",
-      "default": "5",
+      "examples": ["5"],
       "description": "Bugsnag event request frequency."
     },
     "BUGSNAG_ORGANIZATION": {
       "type": "string",
-      "default": "bugsnag-organization-id",
+      "examples": ["bugsnag-organization-id"],
       "description": "A valid Bugsnag organization id."
     },
     "BUGSNAG_PROJECT_NOTIFIER_API_KEY": {
       "type": "string",
-      "default": "bugsnag-project-api-key-id",
+      "examples": ["bugsnag-project-api-key-id"],
       "description": "A valid Bugsnag project API key."
     },
     "BUGSNAG_REPORTING_WINDOW_IN_DAYS": {
       "type": "integer",
-      "default": "14",
+      "examples": ["14"],
       "description": "Specifies how far in the past events should be retrieved."
     },
     "DEFAULT_USAGE_PLAN_ID": {
       "type": "string",
-      "default": "usage-plan-id",
+      "examples": ["usage-plan-id"],
       "description": "AWS API gateway default usage plan used when creating API keys for API users."
     },
     "MAXIMUM_PERMITTED_MONITORED_TRIPS": {
       "type": "integer",
-      "default": "5",
+      "examples": ["5"],
       "description": "The maximum number of saved monitored trips."
     },
     "MONGO_DB_NAME": {
       "type": "string",
-      "default": "otp_middleware",
+      "examples": ["otp_middleware"],
       "description": "The name of the OTP Middleware Mongo DB."
     },
     "MONGO_HOST": {
       "type": "string",
-      "default": "localhost:27017",
+      "examples": ["localhost:27017"],
       "description": "Mongo host address."
     },
     "MONGO_PASSWORD": {
       "type": "string",
-      "default": "(empty string)",
+      "examples": ["(empty string)"],
       "description": "Mongo DB password"
     },
     "MONGO_PROTOCOL": {
       "type": "string",
-      "default": "mongodb",
+      "examples": ["mongodb"],
       "description": "Mongo DB protocol"
     },
     "MONGO_USER": {
       "type": "string",
-      "default": "(empty string)",
+      "examples": ["(empty string)"],
```

Comment:
Might be good to provide better example for this and password (simply `username` and `password` would be fine).